docs/src/main/asciidoc/gcp-functions-http.adoc

```diff
@@ -140,33 +142,18 @@ public class GreetingRoutes {
 
 == Build and Deploy to Google Cloud
 
-To build your application, you first need to define a packaging of type `uber-jar` via your `application.properties`.
-
-[source]
-----
-quarkus.package.uber-jar=true
-----
+NOTE: Quarkus forces a packaging of type `uber-jar` for your function as Google Cloud Function deployment requires a single JAR.
 
 Package your application using the standard `mvn clean package` command.
-The result of the previous command is a single JAR file inside the `target` repository that contains classes and dependencies of the project.
-
-To deploy your JAR to Google Cloud, you need to pass a directory with only this JAR inside it, to the `gcloud` utility.
-
-So first, create a `deployment` directory and copy the generated artifact inside it.
-
-[source]
-----
-mkdir deployment
-cp target/google-cloud-functions-http-1.0-SNAPSHOT-runner.jar deployment/
-----
+The result of the previous command is a single JAR file inside the `target/deployment` repository that contains classes and dependencies of the project.
 
 Then you will be able to use `gcloud` to deploy your function to Google Cloud.
 
 [source]
 ----
 gcloud beta functions deploy quarkus-example-http \
   --entry-point=io.quarkus.gcp.functions.http.QuarkusHttpFunction \
-  --runtime=java11 --trigger-http --source=deployment
+  --runtime=java11 --trigger-http --source=target\deployment
```

Comment:
just to be sure, it's normal it's a `\` and not a `/`?

Reply:
No, I'll change it

Reply:
done